src/Raven.Studio/typescript/viewmodels/database/documents/editDocumentConnectedDocuments.ts

```diff
@@ -151,7 +155,11 @@ class connectedDocuments {
                 "Delete",
                 `<i class="icon-trash"></i>`,
                 "35px",
-                { title: () => 'Delete attachment', extraClass: () => 'file-trash' })
+                {
+                  extraClass: () => 'file-trash',
+                  title: () => 'Delete attachment',
+                  hide: () => this.isReadOnlyAccess()
```

Comment:
hmm I wonder if we actually should call that allowed, since for hide/show we have property: nav. 
We should discuss that.

Reply:
as discussed - leaving this as is